Apps/Examples/Examples/All Examples/OfflineManagerExample.swift

```diff
@@ -124,7 +124,7 @@ final class OfflineManagerExample: UIViewController, ExampleProtocol {
 
         // Load the tile region
         let tileRegionLoadOptions = TileRegionLoadOptions(
-            geometry: MapboxCommon.Geometry(coordinate: tokyoCoord),
+            geometry: MapboxCommon.Geometry(point: tokyoCoord as NSValue),
```

Comment:
we typically convert the CLLocationCoordinate to a CGPoint and then to NSValue. I think that will yield different results than this since the lat/lng are reversed.

Regardless, the need for this workaround will be short-lived since we're in the process of refining this interface to work with Turf.

Reply:
Nevermind, I looked at this some more, and I think it works as expected. Our implementation doesn't actually flip lat/lng in this case.